qlora.py

```diff
@@ -303,6 +334,10 @@ def get_accelerate_model(args, checkpoint_dir):
             print('='*80)
             print('Your GPU supports bfloat16, you can accelerate training with the argument --bf16')
             print('='*80)
+            
+    if compute_dtype == torch.float16 and (is_ipex_available() and torch.xpu.is_available()):
+        compute_dtype = torch.bfloat16
+        print('Intel XPU does not supports float16 yet, you can accelerate training with the argument --bf16')
```

Comment:
Thanks for the PR!
Does this mean that you are effectively switching to bf16 but printing a message that says the user should be switching?
If so, maybe change the message saying "Intel XPU does not support float16 yet, so switching to bfloat16".

Reply:
Thanks for the pointer , modified it.